src/EditorFeatures/Test2/Diagnostics/AbstractCrossLanguageUserDiagnosticTest.vb

```diff
@@ -61,7 +56,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics
                 onAfterWorkspaceCreated?.Invoke(workspace)
 
                 Dim diagnosticAndFix = Await GetDiagnosticAndFixAsync(workspace)
-                Dim codeActions = diagnosticAndFix.Item2.Fixes.Select(Function(f) f.Action).ToList()
+                Dim codeActions As IList(Of CodeAction) = diagnosticAndFix.Item2.Fixes.Select(Function(f) f.Action).ToList()
```

Comment:
surprising.  why is this needed?

Reply:
The line right after assigns `IList(Of CodeAction)`